main.go

```diff
@@ -203,10 +231,31 @@ func main() {
 		os.Exit(1)
 	}
 
+	for _, tp_file := range flagProviders {
+		f, err := os.Open(tp_file)
+		if err != nil {
+			log.Println("couldn't open config file", *flagConfigFile, err)
+			os.Exit(1)
+		}
+		var conf TPConfig
+		dec := yaml.NewDecoder(f)
+		err = dec.Decode(&conf)
+		if err != nil {
+			log.Println("couldn't decode trusted provider config file", *flagConfigFile, err)
+			os.Exit(1)
+		}
+
+		for _, node := range conf.Nodes {
+			cfg.CustomNodes = append(cfg.CustomNodes, node)
+		}
+	}
+
 	// Special traffic class for unstaked nodes visible in gossip (e.g. RPC)
 	cfg.UnstakedClass.Stake = -1
 	cfg.Classes = append(cfg.Classes, cfg.UnstakedClass)
 
+	cfg.CustomNodeClass.Stake = 100
+	cfg.Classes = append(cfg.Classes, cfg.CustomNodeClass)
```

Comment:
Maybe this could be a constant instead of a [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming))?